bitnami/kube-prometheus/templates/exporters/kubelet/servicemonitor.yaml

```
@@ -64,14 +69,22 @@ spec:
       tlsConfig:
         insecureSkipVerify: false
       {{- end }}
-      honorLabels: true
-      {{- if .Values.kubelet.serviceMonitor.interval }}
-      interval: {{ .Values.kubelet.serviceMonitor.interval }}
       {{- end }}
       {{- if .Values.kubelet.serviceMonitor.cAdvisorMetricRelabelings }}
       metricRelabelings: {{- include "common.tplvalues.render" ( dict "value" .Values.kubelet.serviceMonitor.cAdvisorMetricRelabelings "context" $) | nindent 8 }}
       {{- end }}
       {{- if .Values.kubelet.serviceMonitor.cAdvisorRelabelings }}
       relabelings: {{- include "common.tplvalues.render" ( dict "value" .Values.kubelet.serviceMonitor.cAdvisorRelabelings "context" $) | nindent 8 }}
       {{- end }}
+      {{- if .Values.kubelet.serviceMonitor.resource }}
+    - port: {{ printf "%s-metrics" $scheme }}
+      scheme: {{$scheme}}
+      path: {{ .Values.kubelet.serviceMonitor.resourcePath }}
```

Please include tls config:
```suggestion
      path: {{ .Values.kubelet.serviceMonitor.resourcePath }}
      {{- if .Values.kubelet.serviceMonitor.https }}
      tlsConfig:
        caFile: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
        serverName: kubernetes
        insecureSkipVerify: true
      bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
      {{- else }}
```